stencil-library/storybook-pages/angular.stories.mdx

```diff
@@ -0,0 +1,189 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import { SummaryElement } from './utils.tsx';
+import CSSVars from './css-variables.tsx';
+import dedent from 'ts-dedent';
+
+<Meta title="Angular" />
+
+# Justifi Angular web components
+
+# Index
+
+- [Usage](#usage)
+- [Integration Steps](#integration-steps)
+- [Props and Event Handling](#props-and-event-handling)
+- [Calling methods](#calling-methods)
+- [Styling](#styling)
+  - [Components styled with variables](#components-styled-with-variables)
+  - [Styling exported parts](#styling-exported-parts)
+
+# Usage
+
+To integrate Justifi web components into your Angular application, use the `@justifi/webcomponents` package directly. This approach allows you to utilize the standard web components in an Angular environment.
+
+## Integration Steps
+
+1. Include Web Components in index.html:
+
+<Source
+  language="bash"
+  dark
+  code={dedent(`
+    <head>
+      <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/@justifi/webcomponents@4.6.0/dist/webcomponents/webcomponents.css" />
+      <script type='module'
+        src='https://cdn.jsdelivr.net/npm/@justifi/webcomponents@4.6.0/dist/webcomponents/webcomponents.esm.js'></script>
+    </head>
+  `)}
+/>
+
+2. Update Angular Module Configuration:
```

Comment:
I think it's helpful to call out specifically what needs to be updated in the module configuration, which I believe is just adding the `CUSTOM_ELEMENTS_SCHEMA` to `schemas` (so that devs don't have to visually compare to what they have already)

Reply:
Nice suggestion!